twake/frontend/src/app/features/users/api/online-user-realtime-api-client.ts

```diff
@@ -17,12 +17,14 @@ export type SubscribeReturn = {
 export type OnlineUserRealtimeAPIType = {
   getUserStatus: (id: GetUserRequestType) => Promise<GetUserResponseType>;
   getUsersStatus: (ids: GetUsersRequestType) => Promise<GetUsersResponseType>;
+  setUserStatus: (id: GetUserRequestType) => Promise<GetUserResponseType>;
+  setUsersStatus: (ids: GetUsersRequestType) => Promise<GetUsersResponseType>;
   subscribe: (onUpdate: (users: RealtimeUpdateMessageType) => void) => SubscribeReturn;
 };
 
 export const ONLINE_ROOM = '/users/online';
```

Comment:
This should be changed both in backend and frontend to `/users/online/{company_id}`. You can convert this to a function like this on both backend and frontend:
```
export const ONLINE_ROOM = (companyId: string) => `/users/online/${companyId}`;
```